includes/admin_import.php

```diff
@@ -160,23 +160,29 @@ function tsml_import_page()
         if (!empty($_POST['tsml_add_data_source']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
 
             //sanitize URL, name, parent region id, and Change Detection values
-            $data_source_url = trim(esc_url_raw($_POST['tsml_add_data_source'], array('http', 'https')));
+            $data_source_url = $imported_data_source_url = trim(esc_url_raw($_POST['tsml_add_data_source'], array('http', 'https')));
```

Comment:
`$imported_data_source` is there because i wanted to save the actual sheet URL that the user entered as the feed url, but modify the url that gets imported below